src/main/java/com/google/gcloud/examples/StorageExample.java

```diff
@@ -217,13 +219,14 @@ public void run(StorageService storage, Tuple<Path, Blob> tuple) throws Exceptio
     }
 
     @Override
-    Tuple<Path, Blob> parse(String... args) {
+    Tuple<Path, Blob> parse(String... args) throws IOException {
       if (args.length < 2 || args.length > 3) {
         throw new IllegalArgumentException();
       }
       Path path = Paths.get(args[0]);
+      String contentType = Files.probeContentType(path);
```

Comment:
This is part of the example (prod code does not depend on it). I tried it before with zip, jar, xml and text and it was working fine. Per javadoc, null as a default should be fine.